Products/ZPerFactMods/dbutilsPermissions.py

```diff
@@ -26,6 +26,7 @@ def edit_docs(self):
         ('zDB_Delete_Records', 'delete'),
         ('zDB_Update_Records', 'update'),
         ('zDB_Insert_Records', 'insert'),
+        ('zDB_Export_Records', 'export'),
```

Comment:
The second part of the tuple must be the name of a function that is defined above (like `update` etc.). It can be a dummy function like the others, but it must be present.